runtime/compiler/control/CompilationThread.cpp

```diff
@@ -1094,7 +1094,7 @@ void TR::CompilationInfo::setAllCompilationsShouldBeInterrupted()
 
 bool TR::CompilationInfo::useSeparateCompilationThread()
    {
-#if (defined(TR_HOST_X86) || defined(TR_HOST_S390) || (defined(TR_HOST_POWER)) || defined(TR_HOST_ARM))
+#if (defined(TR_HOST_X86) || defined(TR_HOST_S390) || defined(TR_HOST_POWER) || defined(TR_HOST_ARM) || defined(TR_HOST_ARM64))
```

Comment:
This covers all architectures now.  @mpirvu, can we simply remove the `#ifdefs` now and have this the default?

Reply:
Yes, we should remove the `#ifdefs`

Reply:
OK.  @knn-k could you remove the `#ifdefs` please?  These code changes look fine otherwise.